auth/server/src/main/java/com/seagates3/authorization/Authorizer.java

```diff
@@ -79,7 +79,47 @@ ServerResponse authorize(Requestor requestor,
     ServerResponse serverResponse = null;
     AuthorizationResponseGenerator responseGenerator =
         new AuthorizationResponseGenerator();
-    String mainOperation = requestBody.get("S3Action");
+    String s3Operation = requestBody.get("S3Action");
```

Comment:
Not very comfirtable with name of the function
authorize should return true or false. As this is a old function i am ok with it, but if possible check if we can refactor it

Reply:
Not touching the old code now.
and its not isAuthorized(). It is just authorize so this returns response